.ruby-version

```diff
@@ -1 +1 @@
-2.7
+2.7.4
```

Comment:
This was intentional, the intent was to allow the version to be "latest 2.7.x series" on Github / RVM. Does this not work?

This made a huge difference in site build time for the marketing site on netlify, not sure if it has the similarly profound improvement on GH Pages where the dev docs live.

Reply:
Ah, I didn't realize that, will revert

Reply:
(I had changed it because when i went to `bundle install` locally it treated this as "use ruby 2.7.0", which I do not have installed)